docs/js/main.js

```diff
@@ -144,112 +195,532 @@ function versionCompare(base, target) { // Return -1, 0, 1
-          if (sideNavItem)
-            sideNavItem.style.display = null;
-          count++;
-        } else {
+        })
+      count++;
+    } else {
+      e.style.display = "none";
+      if (sideNavItem)
+        sideNavItem.forEach(e => {
           e.style.display = "none";
-          if (sideNavItem)
-            sideNavItem.style.display = "none";
-        }
+        })
+    }
 
-        pass = false; // reset
-      })
+    pass = false; // Reset
+  })
 
-      if (count == 0) {
-        if (document.getElementById("no-matches") == null)
-        document.getElementById("content").insertAdjacentHTML('beforeend', '<p id="no-matches" style="text-align: center;">No matches found.</p>');
-      } else {
-        if (document.getElementById("no-matches") != null)
-          document.getElementById("no-matches").remove();
-      }
-  
-      count = 0; // reset
-    }, 100); // Spam delay for better performance
+  searchResultBox = document.getElementById("search-bar-after");
+  if (count > 0 && (version != "" || searchValue != "" || filterType || filterNew)) {
+    searchResultBox.textContent = `${count} ${resultsFoundText}`
+    searchResultBox.style.display = null;
+  } else {
+    searchResultBox.style.display = "none";
+  }
+
+  if (count == 0) {
+    if (document.getElementById("no-matches") == null)
+    document.getElementById("content").insertAdjacentHTML('beforeend', '<p id="no-matches" style="text-align: center;">No matches found.</p>');
+  } else {
+    if (document.getElementById("no-matches") != null)
+      document.getElementById("no-matches").remove();
+  }
 
-  }); 
+  count = 0; // reset
 }
-// Search Bar </>
 
-// <> HighlightJS 
-document.querySelectorAll('pre.code').forEach(el => { // Apply the code formatting on the same <pre> not the <code> inside to not break the styling
-  hljs.highlightElement(el);
-});
-document.querySelectorAll('div .skript-code-block').forEach(el => { // This lags the docs pages due to the huge amount of elements being parsed, we can disable this if lag is so bad for some people or keep it because it looks AMAZING!
-  hljs.highlightElement(el);
-});
-// HighlightJS </>
+if (searchBar) {
+  searchBar.focus() // To easily search after page loading without the need to click
+  searchBar.addEventListener('keydown', (event) => {
+    setTimeout(() => { // Important to actually get the value after typing or deleting + better performance
+      searchNow();
+    }, 100);
+  });
+}
+// Search Bar </>
 
 // <> Placeholders
-function replacePlaceholders(html) {
-  let innerHTML = html.innerHTML;
+
+function replacePlaceholders(element) {
+  let innerHTML = element.innerHTML;
   if (innerHTML.includes("${latest-version}")) {
-    let lv = $.getJSON("https://api.github.com/repos/SkriptLang/Skript/releases?per_page=1", (data) => {
-      html.innerHTML = html.innerHTML.replaceAll("${latest-version}", data[0]["tag_name"]);
-    })
+    getApiValue(element, "ghapi-latest-version", "releases", (data) => {
```

Comment:
How will this (and rest of the site relying on GH versions) behave if we, for example, release a 2.6.4 after 2.7?

Reply:
Will probably show 2.6.4 which is a problem

Reply:
I might have a better solution for this but will discuss it on discord later
Task for another PR (Leave unresolved)

Reply:
I don't know if this was discussed already but I'd like to add that GitHub recently released a feature that allows [explicitly setting the latest release](https://github.blog/changelog/2022-10-21-explicitly-set-the-latest-release). I would guess that this would be a better solution than manually checking for the latest version.
In the case that this feature is used, the `releases/latest` path should be used instead of `releases`.

Reply:
This is actually a decently old feature, but it's a great idea.

Reply:
That seems to be a great idea indeed, I will check it out